### PR TITLE
fix(cert script): remove sudo and streamline script

### PR DESCRIPTION
**Pull request by ferantivero**

### CODE REVIEW

File: certs/letsencrypt-pip-cert-generation.sh [hunk 1]
```diff
@@ -27,13 +27,10 @@ az storage blob upload \
...
+certbot certonly --manual --manual-auth-hook "${DIR}/authenticator.sh $STORAGE_ACCOUNT_NAME $STORAGE_CONNECTION_STRING" -d $FQDN --config-dir ./certs/output/etc/letsencrypt --work-dir ./certs/output/var/lib/letsencrypt --logs-dir ./certs/output/var/log/letsencrypt
 
-sudo cp /etc/letsencrypt/live/$FQDN/privkey.pem .
-sudo cp /etc/letsencrypt/live/$FQDN/cert.pem .
-sudo cp /etc/letsencrypt/live/$FQDN/chain.pem .
-openssl pkcs12 -export -out $SUBDOMAIN.pfx -inkey privkey.pem -in cert.pem -certfile chain.pem -passout pass:
+openssl pkcs12 -export -out $SUBDOMAIN.pfx -inkey ./certs/output/etc/letsencrypt/live/$FQDN/privkey.pem -in ./certs/output/etc/letsencrypt/live/$FQDN/cert.pem -certfile ./certs/output/etc/letsencrypt/live/$FQDN/chain.pem -passout pass:
 
 echo "Deleting resources"
 az group delete -n $RGNAME --yes
```
Comment (neilpeterson) [inline certs/letsencrypt-pip-cert-generation.sh:35]: This operation is lengthy, consider adding `--no-wait` to help speed up the process.

**Review by neilpeterson — APPROVED** [review]
@ferantivero updated script runs great on my mac, thanks for the updates.

**Comment by ferantivero** [comment]
w00t - glad to hear that @neilpeterson it is now merged 💪

File: certs/letsencrypt-pip-cert-generation.sh (hunk 1 above)
Comment (ckittel) [reply certs/letsencrypt-pip-cert-generation.sh:35]: This as been done and merged in.

**Pull request merged**